### ISSUE TITLE
Replace current `testnet` deployment with new k8s deployment

### ISSUE THREAD

**Issue by conorsch** [penumbra-zone/penumbra#1818]
Follow-up to #1659. 

We want to replace completely the legacy testnet infra with the k8s deployment scripts used for testnet-preview. We tried to do this for `039-praxidike` (#1757), but encountered a regression that blocked use of the testnet (https://github.com/penumbra-zone/penumbra/issues/1815). Here's what's blocking vigorous testing of the new setup:

- [x] https://github.com/penumbra-zone/penumbra/issues/1815 [regression]
- [x]  https://github.com/penumbra-zone/penumbra/issues/1816
- [ ] https://github.com/penumbra-zone/penumbra/issues/1832 (nice-to-have)
Once those are done, we'll see the results immediately on preview, and can commence interactive testing to ensure it meets our expectations—before it's labeled for a release.

**Comment by conorsch** [comment]
I'll follow up with a step-by-step checklist for cut-over day-of, as was done in #1659.

**Comment by conorsch** [comment]
Local smoke-testing targeting devnets based on latest main are looking good. I'd like to get #1659 in today, and thereafter begin on the standard early steps for testnet prep, such as bumping the chain id (#1814). Once those changes land, I'll invite the rest of the team to bang on preview and see what shakes out.

**Comment by conorsch** [comment]
Updated chain id. That completes the outstanding changes, and we're now ready for testing on `testnet-preview.penumbra.zone`. Please have it, folks, and report any issues you encounter.

**Comment by conorsch** [comment]
As of 040-themisto (#1814), the public testnet is based on k8s, same as preview. We'll continue to iterate on the config, but we'll track that work in other issues.
